src/components/add-form/index.js

```diff
@@ -1,28 +1,40 @@
 import React from 'react';
 import './styles.css';
+import { FiX } from 'react-icons/fi';
 
 const TimeFrames = {
   soon: '7',
   kindOfSoon: '14',
   notSoon: '30',
 };
 
-const AddForm = ({ formState, handleForm }) => (
+const AddForm = ({ formState, handleForm, setDefaultValues }) => (
   <div className="addform-container">
-    <label htmlFor="productName" className="addform-container-label">
-      Item name:
-      <input
-        className="input-itemform"
-        id="productName"
-        name="productName"
-        type="text"
-        maxLength="30"
-        placeholder="Write here the item to add..."
-        value={formState.productName}
-        onChange={(event) => handleForm(event)}
-        required
-      />
-    </label>
+    <div className="addform-inputgroup">
+      <label htmlFor="productName" className="addform-container-label">
+        Item name:
+        <input
+          className="input-itemform"
+          id="productName"
+          name="productName"
+          type="text"
+          maxLength="30"
+          placeholder="Write here the item to add..."
+          value={formState.productName}
+          onChange={(event) => handleForm(event)}
+          required
+        />
```

Comment:
We could put an `aria-label` on this input too 😃

Reply:
done ^^ thanks for the suggestion!